.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -314,3 +316,4 @@ jobs:
         sycl_cts_artifact: ${{ inputs.sycl_cts_artifact }}
         target_devices: ${{ inputs.target_devices }}
         retention-days: ${{ inputs.retention-days }}
+        tests_ref: ${{ inputs.tests_ref }}
```

Comment:
Needs more alignment with e2e still. Something has to be done with line 302. I think the `devops/actions/run-tests/*` should accept single `ref` input (as in line 302). And the job of the `.yml` file is to translate `tests_ref`/`ref`/etc. into such an argument.

Reply:
Hm, right, likely we don't need a new parameter, we can just use ref for this purpose.

Reply:
Do I get it right? 28c7483e3091fb123966a7c2c6dbeefab8f5c8ef

Reply:
I'd prefer

```
ref:  #maybe rename to llvm_ref/repo_ref/toolchain_ref or something
tests_ref:
  required: False  # no default
  type: string

jobs:
...
   e2e:
     ref: ${{ tests_ref || ref || "main" }}

   cts:
     ref: ${{ tests_ref || "main" }}
```